library/src/main/java/com/schibsted/spain/barista/custom/ClickChildAction.java

```diff
@@ -0,0 +1,61 @@
+package com.schibsted.spain.barista.custom;
+
+import android.support.test.espresso.UiController;
+import android.support.test.espresso.ViewAction;
+import android.view.View;
+import android.widget.TextView;
+
+import com.schibsted.spain.barista.util.ViewTreeAnalyzer;
+
+import org.hamcrest.Matcher;
+
+public class ClickChildAction {
+
+  public static ViewAction clickChildWithId(final int id) {
```

Comment:
About the first, well catched! Going to do it!
About the second, it's not needed at the moment. Nice to have buuuut probably we could do it in another PR...